frontend/src/metabase/home/components/CustomHomePageModal/CustomHomePageModal.unit.spec.tsx

```diff
@@ -1,54 +1,18 @@
-import userEvent from "@testing-library/user-event";
 import { renderWithProviders, screen } from "__support__/ui";
 
-import {
-  setupCollectionsEndpoints,
-  setupCollectionItemsEndpoint,
-  setupDashboardEndpoints,
-} from "__support__/server-mocks";
-
-import {
-  createMockCollection,
-  createMockCollectionItem,
-  createMockDashboard,
-} from "metabase-types/api/mocks";
 import { CustomHomePageModal } from "./CustomHomePageModal";
 
-const ROOT_COLLECTION = createMockCollection({
-  id: "root",
-  can_write: true,
-  name: "Our Analytics",
-  location: undefined,
-});
-const COLLECTION_ITEM = createMockCollectionItem({
-  name: "Foo",
-  model: "dashboard",
-});
-
-const FOO_DASHBOARD = createMockDashboard({
-  name: "Foo",
-});
-
 const setup = ({ ...props } = {}) => {
   const onClose = jest.fn();
 
-  setupCollectionsEndpoints({ collections: [ROOT_COLLECTION] });
-  setupCollectionItemsEndpoint(ROOT_COLLECTION, [COLLECTION_ITEM]);
-  setupDashboardEndpoints(FOO_DASHBOARD);
-
   renderWithProviders(
     <CustomHomePageModal onClose={onClose} isOpen={true} {...props} />,
   );
 };
 
 describe("custom hompage modal", () => {
-  it("should only enable the save button if a dashboard has been selected", async () => {
+  it("should only enable the save button if a dashboard has been selected", () => {
```

Comment:
wow, no need for setup a endpoints? don't you have warnings in console from fetch-mock?

Reply:
Nope! I'm assuming because we never actually render the item picker, it doesn't end up making an API call. In theory we are trying to load a dashboard, but since we're passing undefined for the ID, I'm guessing we never actually make a request for that either.